pyriksdagen/utils.py

```diff
@@ -335,3 +333,7 @@ def get_data_location(partition):
     d["motions"] = os.environ.get("MOTIONS_PATH", "data")
     d["metadata"] = os.environ.get("METADATA_PATH", "data")
     return d[partition]
+
+
+def where_am_i():
```

Comment:
What is this for?

Reply:
need it to fetch the queries

Reply:
![image](https://github.com/swerik-project/pyriksdagen/assets/8887511/ece3eef8-7150-4b74-b18d-43c3c3120325)

Reply:
[52721c1](https://github.com/swerik-project/pyriksdagen/pull/1/commits/52721c1e18f9c4afafd5cfacadebcad86666f0c2)